index.html

```diff
@@ -24,6 +24,11 @@
       <a href="#" id="logo">My logo</a>
       <nav class="nav">
         <img class="union" src="images/Union.png" alt="img" />
+        <ul class="top_navbar">
+          <li class="navbar_li">Portfolio</li>
+          <li class="navbar_li">About</li>
+          <li class="navbar_li">Contact</li>
```

Comment:
- [ ] please like the previous reviewer noted, some buttons in the mobile version still have this 👇🏾 background color. kindly change them
![Screenshot (927)](https://user-images.githubusercontent.com/65298048/229269512-138ae082-3593-4d14-93f9-42296b640eea.png)

required version
![Screenshot (928)](https://user-images.githubusercontent.com/65298048/229269546-74cbf5e7-184d-465e-aa28-033a7ec16215.png)